1.table.sql

```diff
@@ -0,0 +1,42 @@
+use CostumeDB
+go
+drop table if exists Sales
+go
+
+create table dbo.Sales(
+    CustomerId int not null identity primary key,
+    CustomerName varchar(50) not null 
+        constraint ck_customer_name_cannot_be_blank check(CustomerName <> ''), 
+    CostumesBought varchar(25) not null
```

Comment:
AS: Take out cannot be blank from the constraint name, since if a costume can only be one of the costumes from the list than automatically it wont be blank.

Reply:
done.